Source/Core/DolphinQt2/MenuBar.cpp

```diff
@@ -173,10 +173,17 @@ void MenuBar::AddViewMenu()
 void MenuBar::AddHelpMenu()
 {
   QMenu* help_menu = addMenu(tr("Help"));
+  QAction* website = help_menu->addAction(tr("Website"));
+  connect(website, &QAction::triggered, this,
+          [=]() { QDesktopServices::openUrl(QUrl(QStringLiteral("https://dolphin-emu.org/"))); });
```

Comment:
I'd guess its copy&paste from the Documentation item, which also has a `=` capture list.